### PR TITLE
error toast

### PR DESCRIPTION
**Pull request by RoriCremer**
Will we have non-error alerts too?

It's not storing the errors--that will need to added as separate storage

dB8u9njEc3

### CODE REVIEW

File: src/components/DatasetPreviewView.jsx [hunk 1]
```diff
@@ -64,18 +64,7 @@ const styles = theme => ({
...
-      '"Segoe UI"',
-      'Roboto',
-      '"Helvetica Neue"',
-      'Arial',
-      'sans-serif',
-      '"Apple Color Emoji"',
-      '"Segoe UI Emoji"',
-      '"Segoe UI Symbol"',
-    ].join(','),
+    fontFamily: theme.typography.fontFamily,
```
Comment (thathert) [inline src/components/DatasetPreviewView.jsx:67]: Thank you.

File: src/components/Toast.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    dispatch(hideAlert());
+  }
+
+  render() {
+    const { classes, errorMsg } = this.props;
+    return (
+      <Card className={classes.card}>
+        <CardContent className={classes.content}>
+          <ErrorIcon className={classes.icon} alt="logo" />
+          <div className={classes.text}>{errorMsg || 'WTF did you do?'}</div>
```
Comment (thathert) [inline src/components/Toast.jsx]: "An error occurred, please try again or submit a bug report." will probably look better in a demo :)

**Review by thathert — APPROVED** [review]
Looks good so far. Approving with expectation that alerts can be dismissed by id and that that message gets updated

File: src/components/Toast.jsx (hunk 2 above)
Comment (RoriCremer) [reply src/components/Toast.jsx]: sorry--it was a Saturday so I couldn't help myself.

**Review by ahaessly — APPROVED** [review]
👍

**Pull request merged**